public_dropin_gpu_environments/triton_server/Dockerfile

```diff
@@ -15,10 +15,6 @@ ENV CODE_DIR=/opt/code ADDRESS=0.0.0.0:8080
 WORKDIR ${CODE_DIR}
 COPY ./*.sh ${CODE_DIR}/
 
-# Triton model configuration parser generated from the protocol buffer scheme
-COPY model_config/ ./model_config/
-ENV PYTHONPATH=$PYTHONPATH:${CODE_DIR}/model_config/
-
```

Comment:
The `tritonclient` already includes the ModelConfig protobuf scheme, so we don't need to maintain it by ourselves.